### DISCUSSION TITLE
Ban/Jail/Mute logging

### DISCUSSION THREAD

**Discussion by amadeusmz** [Zrips/CMI#6912] [Enhancement]
I need a command displaying recent punishments
- PlayerName was banned by amadeusmz for 3 days, reason: cheating, date: 15:00 29/03
- OtherPlayer was jailed by amadeusmz for 3 days, reason: cheating, date: 15:00 29/03
- ...

Also  a paginated hologram displaying this log would be nice
And a command displaying punishment status of a player
/status [player]
player isn't having any punishment at the moment
or
player is currently jailed, time left: 3 days 2 hours, operator: amadeusmz, reason: cheating

**Comment by CloudeLecaw** [comment]
what about the /cmi banlist, /cmi ipballist
/cmi jaillist (jailName) (cellId)


well the only thing that doesnt exist is a mutelist...
what i would suggest here is a new argument for the /cmi mute command -> list
cmi mute [playerName|list] (time) (-s) (reason)]

What the list should do?
Well list all muted players with lengh and reason - current online once prefered to offline once.

about punishmentlist # warninglist ....players can do /cmi wanrings  so see there current warning points

**Comment by amadeusmz** [comment]
I want it to be fully logged as described above, like the list of recent bans of LiteBan

**Comment by CloudeLecaw** [comment]
You could create a custom alias that lists all these information at once via msgs. use he cmi placeholders compares with parse others exansionof papi for this :)

i dont think an extra log for this will be created. since u can view it all seperatly and also took a look on your database allready

**Comment by amadeusmz** [comment]
I already took a look at the database, but it doesn't store who executed the jail/mute command, nor when it was executed.

**Comment by CloudeLecaw** [comment]
your using Discord? u could define a channel there where commands like jail mute ban etc will be listet and send them to a seperate channel that only a few people have access to
